src/components/Button/Button.tsx

```diff
@@ -144,33 +146,39 @@ export type Props = $Omit<React.ComponentProps<typeof Surface>, 'mode'> & {
  * export default MyComponent;
  * ```
  */
-const Button = ({
-  disabled,
-  compact,
-  mode = 'text',
-  dark,
-  loading,
-  icon,
-  buttonColor: customButtonColor,
-  textColor: customTextColor,
-  rippleColor: customRippleColor,
-  children,
-  accessibilityLabel,
-  accessibilityHint,
-  onPress,
-  onPressIn,
-  onPressOut,
-  onLongPress,
-  delayLongPress,
-  style,
-  theme: themeOverrides,
-  uppercase: uppercaseProp,
-  contentStyle,
-  labelStyle,
-  testID = 'button',
-  accessible,
-  ...rest
-}: Props) => {
+
+type ButtonRef = React.ElementRef<typeof TouchableRipple>;
+
+const Button: React.ForwardRefRenderFunction<ButtonRef, Props> = (
```

Comment:
Have you tried something like this?
```suggestion
const Button = forwardRef({
```

Reply:
@DimitarNestorov I could not understand your point.

`forwardRef` is already added in the last line `export default forwardRef(Button);`

Reply:
@DimitarNestorov We cannot use `forwardRef` from `src/utils/forwardRef.tsx` because it does not support mentioning component level attribute,  In our case `TouchableRipple.supported = true;` is a component level attribute.